examples/resources/awscc_chatbot_slack_channel_configuration/first_example.tf

```diff
@@ -0,0 +1,14 @@
+
+resource "awscc_chatbot_slack_channel_configuration" "example" {
+  # Set the name to identify the configuration in the ChatBot console
+  configuration_name = "ChatBot-Example"
+
+  # Replace <account_id> and <role-name> with sutable values
+  iam_role_arn = "arn:aws:iam::<account_id>:role/<role-name>"
+
+  # Replace <channel-id> with the id of the Slack channel you want to send message to
+  slack_channel_id = var.channel-id
+
+  # Replease <workspace-id> with the id of the Slack workspace
+  slack_workspace_id = "var.workspace-id"
+}
```

Comment:
```suggestion
resource "awscc_chatbot_slack_channel_configuration" "example" {
  configuration_name = "example-slack-channel-config"
  iam_role_arn       = awscc_iam_role.example.arn
  slack_channel_id   = var.channel_id
  slack_workspace_id = var.workspace_id
}

resource "awscc_iam_role" "example" {
  role_name = "ChatBot-Channel-Role"
  assume_role_policy_document = jsonencode({
    Version = "2012-10-17"
    Statement = [
      {
        Action = "sts:AssumeRole"
        Effect = "Allow"
        Sid    = ""
        Principal = {
          Service = "chatbot.amazonaws.com"
        }
      },
    ]
  })
  managed_policy_arns = ["arn:aws:iam::aws:policy/AWSResourceExplorerReadOnlyAccess"]
}

```

Reply:
Thanks, yes, that's what I was going to add